.gitignore

```diff
@@ -1,9 +1,13 @@
 
```

Comment:
great idea!

Reply:
Agree re CSV files.  We need to consolidate and streamline the files (and the two classes).